### PR TITLE
Refactor Blogger importer with guard clauses

### PR DESCRIPTION
**Pull request by ashmaroli**
Basically reducing number of lines and code indentation due to nested conditionals

### CODE REVIEW

**Review by DirtyF — APPROVED** [review]

**Review by parkr — APPROVED** [review]
@ashmaroli This looks great! Unfortunately there are some issues with merge conflicts (I merged another PR of yours into this importer). Want to fix those up and merge +minor?

File: lib/jekyll-import/importers/blogger.rb [hunk 1]
```diff
@@ -118,9 +113,7 @@ def tag_start(tag, attrs)
 
             @in_entry_elem = { :meta => {}, :body => nil }
           when "title"
-            if @in_entry_elem
-              raise 'only <title type="text"></title> is supported' if attrs["type"] != "text"
-            end
+            raise 'only <title type="text"></title> is supported' if @in_entry_elem && attrs["type"] != "text"
```
Comment (parkr) [inline lib/jekyll-import/importers/blogger.rb:116]: if we `raise`, will that completely stop the processing of the blogger import?

File: lib/jekyll-import/importers/blogger.rb [hunk 2]
```diff
@@ -267,19 +260,16 @@ def post_data_from_in_entry_elem_info
             { :filename => filename, :header => header, :body => body }
           elsif element_meta[:kind] == "comment"
             timestamp = Time.parse(element_meta[:published]).strftime("%Y-%m-%d")
-            if element_meta[:original_url]
-              @comment_seq ||= 1
+            raise "Original URL is missing" unless element_meta[:original_url]
```
Comment (parkr) [inline lib/jekyll-import/importers/blogger.rb:263]: if we raise here, will we abort the whole import? would it be better to continue with a warning?

Comment (parkr) [reply lib/jekyll-import/importers/blogger.rb:263]: OH I see, this is an original `raise`, not something you added. Sorry!

File: lib/jekyll-import/importers/blogger.rb (hunk 1 above)
Comment (parkr) [reply lib/jekyll-import/importers/blogger.rb:116]: Oh, same here, this was present before your refactor. 😄

**Comment by parkr** [comment]
@jekyllbot: merge +dev

**Pull request merged**